ci/pre_commit_hook_sample

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
```

Comment:
Should add license header

Reply:
Thank you!

Actually it's a fair point of what we can try to add to hook (check that all files (c, cpp,h, sh at least) have the license

Reply:
We normally add the license header in the source files and scripts (shell, cmake, etc.), and don't add it in the document. It should be ok here not to add the license header if we treat it as document and it really hasn't license issue.

Reply:
I've already added it

I'm just curious would we like to have this check in pre-commit hook or not (let's say grep source file for `apache-2.0` to see that license header is added and don't allow to commit without it)

Reply:
Yes, it would be great to have the license header check in pre-commit hook.